tools/validators/instance_validator/error_guide.md

```diff
@@ -0,0 +1,1712 @@
+  translation:
+    return_water_temperature_sensor:
+      present_value: "points.return_water_temperature_sensor.present_value"
+      units:
+        key: "pointset.points.return_water_temperature_sensor.units"
+        values:
+          degrees_celsius: "degC"
+    supply_water_temperature_sensor:
+      present_value: "points.supply_water_temperature_sensor.present_value"
+      units:
+        key: "pointset.points.supply_water_temperature_sensor.units"
+        values:
+          degrees_celsius: "degC"
+    thermal_power_capacity:
+      present_value: "points.thermal_power_capacity.present_value"
+      units:
+        key: "pointset.points.thermal_power_capacity.units"
+        values:
+          kilowatts: "kilowatts"
+    differential_pressure_specification:
+      present_value: "points.differential_pressure_specification.present_value"
+      units:
+        key: "pointset.points.differential_pressure_specification.units"
+        values:
+          pascals: "pascals"
+
+US-SEA-BLDG1-GUID:
+  type: FACILITIES/BUILDING
+  code: US-SEA-BLDG1
+```
+
+### Error Message:
+```
+[ERROR]	Entity VIRTUAL-ENTITY-GUID (VIRTUAL-ENTITY) links to a source entity: PHYSICAL-ENTITY-GUID (PHYSICAL-ENTITY) that does not have the linked source field: return_water_temperature_sensor_200. Check that this field on source translation exists.
+```
+
+## Spacing
+
+### Description:
+Ensure there are no extra or missing leading spaces on a line as YAML files are read based on indentation.
+
+### Example Bad Config:
+```
+US-SEA-BLDG1-GUID:
+  type: FACILITIES/BUILDING
+   code: US-SEA-BLDG1
+```
+
+### Error Message:
+```
+mapping values are not allowed here
+  in "<unicode string>", line 3, column 8:
+       code: US-SEA-BLDG1
+           ^ (line: 3)
+```
+
+## Tabbing
+
+### Description:
+Ensure there are no extra or missing leading tabs on a line as YAML files are read based on indentation.
+
+### Example Bad Config:
+```
+US-SEA-BLDG1-GUID:
+    type: FACILITIES/BUILDING
+  code: US-SEA-BLDG1
+```
+
+### Error Message:
+```
+while parsing a block mapping
+  in "<unicode string>", line 1, column 1:
+    US-SEA-BLDG1-GUID:
+    ^ (line: 1)
+expected <block end>, but found '<block mapping start>'
+  in "<unicode string>", line 3, column 3:
+      code: US-SEA-BLDG1
+      ^ (line: 3)
+```
+
+## All Fields Missing 
+
+### Description:
+Every non-facility entity requires at least one non-missing field.
+
+### Example Bad Config:
+```
+SDC_EXT-17-GUID:
+  type: HVAC/SDC_EXT
+  code: SDC_EXT-17
+  cloud_device_id: "1234567890123456"
+  translation:
+    shade_extent_percentage_command: MISSING
+
+US-SEA-BLDG1-GUID:
+  type: FACILITIES/BUILDING
+  code: US-SEA-BLDG1
+```
+
+### Error Message:
+```
+[ERROR]	Entity SDC_EXT-17-GUID (SDC_EXT-17) has all field translations marked as MISSING. This is not allowed.
+```
+
+## Translation Compliance
+
+### Description:
+Everything given in the translation section of a config must match, case-sensitive, to what is in DBO.
```

Comment:
Expand description here -- field names and DBO units are all lower-case, type names and DBO states are upper-case